ui/src/main/java/org/pentaho/di/ui/core/widget/tree/LeveledTreeNode.java

```diff
@@ -0,0 +1,94 @@
+/*! ******************************************************************************
+ *
+ * Pentaho
+ *
+ * Copyright (C) 2024 by Hitachi Vantara, LLC : http://www.pentaho.com
+ *
+ * Use of this software is governed by the Business Source License included
+ * in the LICENSE.TXT file.
+ *
+ * Change Date: 2029-07-20
+ ******************************************************************************/
+
+
+package org.pentaho.di.ui.core.widget.tree;
+
+import java.util.Comparator;
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.graphics.Device;
+import org.eclipse.swt.widgets.Display;
+import org.eclipse.swt.widgets.TreeItem;
+import org.pentaho.di.i18n.BaseMessages;
+import org.pentaho.di.ui.spoon.Spoon;
+
+public class LeveledTreeNode extends TreeNode {
+  public enum LEVEL {
+    DEFAULT( 1 ),
+    PROJECT( 2 ),
+    GLOBAL( 3 ),
```

Comment:
Based  off the wiki is GLOBAL suppose to be SYSTEM ?

https://hv-eng.atlassian.net/wiki/spaces/ENG/pages/31925961450/Project+Profile+Feature+Overview#Levels